docs/content/configuration/broker.md

```diff
@@ -45,6 +45,8 @@ Druid uses Jetty to serve HTTP requests.
 |`druid.broker.http.compressionCodec`|Compression codec the Broker uses to communicate with historical and real-time processes. May be "gzip" or "identity".|gzip|
 |`druid.broker.http.readTimeout`|The timeout for data reads from historical and real-time processes.|PT15M|
 |`druid.server.http.maxQueryTimeout`|Maximum allowed value (in milliseconds) for `timeout` parameter. See [query-context](query-context.html) to know more about `timeout`. Query is rejected if the query context `timeout` is greater than this value. |Long.MAX_VALUE|
+|`druid.server.http.maxRequestHeaderSize`|Maximum size of a request header in bytes. Larger headers consume more memory and can make a server more vulnerable to denial of service attacks. |8 * 1024|
```

Comment:
is there recommendation on the max possible value that user should set ?

Reply:
generally default settings would work for most of the use cases, Till now I have seen only the Kerberos request header going beyond that in case of Active Directory setup where the recommendation is to set it to  4/3 * T, where T is the user's token size in bytes. http://support.microsoft.com/en-in/help/2020943/http-400-bad-request-request-header-too-long-response-to-http-request

Reply:
alright.